js/gomodule.go

```diff
@@ -0,0 +1,93 @@
+package js
+
+import (
+	"github.com/dop251/goja"
+	"go.k6.io/k6/js/modules"
+)
+
+// baseGoModule is a go module that does not implement modules.Module interface
+// TODO maybe depracate those in the future
+type baseGoModule struct {
+	mod interface{}
+}
+
+var _ module = &baseGoModule{}
+
+func (b *baseGoModule) Instantiate(vu modules.VU) moduleInstance {
+	return &baseGoModuleInstance{mod: b.mod, vu: vu}
+}
+
+type baseGoModuleInstance struct {
+	mod      interface{}
+	vu       modules.VU
+	exportsO *goja.Object // this is so we only initialize the exports once per instance
```

Comment:
You have a `moduleObj` in the other `module` implementation, so this should probably be `exportsObj`. Or, better yet, remove the `O` and `Obj` from both and just have `module` and `exports`?

Reply:
It can't be `exports` as `exports` is a method on the interface. 

I guess I can make the interface method upperCase as the intention is that this will be ripped out in a separate package in the future :thinking:

Reply:
:thinking: ignore this then, it's a minor nitpick all things considered